buf/registry/module/v1beta1/label_service.proto

```diff
@@ -96,7 +96,9 @@ message ListLabelsRequest {
   // Once the resource is resolved, the following Labels are listed:
   //   - If a Module is referenced, all Labels for the Module are returned.
   //   - If a Label is referenced, this Label is returned.
-  //   - If a Commit is referenced, all Labels for the Commit are returned.
+  //   - If a Commit is referenced, all Labels for the Commit are returned. Note that this will
+  //     not return any Labels where the Commit is currently rejected or pending, it will only
+  //     return Labels for Commits that are passed or accepted.
```

Comment:
What about blocked commits? I imagine we want to treat them similar to pending?

Reply:
yes, I pushed a f3d9cb2f024fdc12041b7e0550b7a8eba919b2c0 to update docs